lasagne/layers/merge.py

```diff
@@ -91,7 +128,7 @@ def get_output_for(self, inputs, **kwargs):
             if coeff != 1:
                 input *= coeff
             if output is not None:
-                output += input
+                output = self.merge_function(output, input)
             else:
                 output = input
         return output
```

Comment:
I wouldn't duplicate the merge functionality here, but just use the function I suggested: https://github.com/Lasagne/Lasagne/pull/301#issuecomment-112519950